README.md

```diff
@@ -1,79 +1,87 @@
+- Add the `UIView+AutoLayoutHelper.swift` file to your Xcode project.
+- **Note**. CocoaPods support and Dynamic Framework target both coming soon!
 
 ### Usage
 
-Examples:
+#### Examples:
 
-Add NSLayoutConstraint relations for a UIView relating its left, top, bottom and right edges to it's superview 
+##### Fill superview
+
+Add `NSLayoutConstraint` relations for a `UIView` relating its top, leading, trailing and bottom edges to it's superview 
 
     // Create view
     
-    let leftView: UIView = UIView(frame: CGRectZero)
+    let leftView = UIView(frame: CGRectZero)
     leftView.backgroundColor = UIColor.redColor()
     self.view.addSubview(leftView)
     
     // Add constraints
-    
-    if let superview = leftView.superview {
         
-        leftView.setTranslatesAutoresizingMaskIntoConstraints(false)
-        
-        leftView.addTopConstraint(toView: superview, attribute: NSLayoutAttribute.Top, relation: NSLayoutRelation.Equal, constant: 10.0)
-        leftView.addLeftConstraint(toView: superview, attribute: NSLayoutAttribute.Left, relation: NSLayoutRelation.Equal, constant: 10.0)
-        leftView.addRightConstraint(toView: superview, attribute: NSLayoutAttribute.Right, relation: NSLayoutRelation.Equal, constant: -10.0)
-        leftView.addBottomConstraint(toView: superview, attribute: NSLayoutAttribute.Bottom, relation: NSLayoutRelation.Equal, constant: -10.0)
-    }
-
-or shorter assuming top to top, left to left, right to right and bottom to bottom you can omit the attribute: 
+    leftView.setTranslatesAutoresizingMaskIntoConstraints(false)
     
-    leftView.addTopConstraint(toView: superview, relation: NSLayoutRelation.Equal, constant: 10.0)
-    leftView.addLeftConstraint(toView: superview, relation: NSLayoutRelation.Equal, constant: 10.0)
-    leftView.addRightConstraint(toView: superview, relation: NSLayoutRelation.Equal, constant: -10.0)
+    leftView.addTopConstraint(toView: superview, attribute: NSLayoutAttribute.Top, relation: NSLayoutRelation.Equal, constant: 10.0)
+    leftView.addLeadingConstraint(toView: superview, attribute: NSLayoutAttribute.Leading, relation: NSLayoutRelation.Equal, constant: 10.0)
+    leftView.addTrailingConstraint(toView: superview, attribute: NSLayoutAttribute.Trailing, relation: NSLayoutRelation.Equal, constant: -10.0)
     leftView.addBottomConstraint(toView: superview, attribute: NSLayoutAttribute.Bottom, relation: NSLayoutRelation.Equal, constant: -10.0)
 
-or even more succinctly:
+or shorter you can omit the attributes:
+    
+    leftView.addTopConstraint(toView: superview, constant: 10.0)
+    leftView.addLeadingConstraint(toView: superview, constant: 10.0)
+    leftView.addTrailingConstraint(toView: superview, constant: -10.0)
+    leftView.addBottomConstraint(toView: superview, constant: -10.0)
 
-    [leftView fillSuperview:UIEdgeInsetsMake(10.0, 10.0, 10.0, 10.0)];
+or even shorter using `fillSuperview` (via [@danieladias](https://github.com/danieladias) )
```

Comment:
The method is `fillSuperView` so I'll update the doc to reflect this because changing the method name should not be part of this pull request (since that would be a breaking change to the API)